### ISSUE TITLE
[$20][MSFT-143] The NVDA doesn't read link in sub-menu when user try to mouse hover on them

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2836]
**Steps to Reproduce**
1. Navigate to https://www.topcoder.com/challenges?
2. Try to mouse hover on primary header menu: Compete/ Tracks/ Community
3. Then try to mouse hover on sub-menu of them

**Expected Result**
The NVDA should read link in sub-menu when user try to mouse hover on them

**Actual Result**
The NVDA doesn't read link in sub-menu when user try to mouse hover on them

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.2.3 Audio Description or Media Alternative (Prerecorded)
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: 

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F15

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096040 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096040 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096040 has been updated - it has been assigned to gets0ul.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097133 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097133 has been updated - it has been assigned to gets0ul.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
Page : https://test-community-app.topcoder-dev.com/challenges

**Comment by tosha5252** [comment]
Not working in Chrome browser.

Works in Edge browser.

Operating System: Windows 10

**Comment by sushilshinde** [comment]
Code is present there on the production 

https://github.com/topcoder-platform/community-app/pull/2977/files

https://github.com/topcoder-platform/community-app/blob/894e4486f05201b26ffa041cb1d8e5d502eb07f9/src/shared/components/TopcoderHeader/desktop/SubMenu/Item/index.jsx#L21

**Comment by gets0ul** [comment]
This is tested on local production. Chrome. Windows 10  
  
https://drive.google.com/file/d/12q8B1p2gYqymrIskj50ywl-anWbkPjHh/view?usp=sharing

**Comment by lakshmiathreya** [comment]
Working on Win/Chrome Jaws reads on mouse over of Sub menu items. @tosha5252 pls double check. fyi - @nkumar-topcoder @sushilshinde

**Comment by tosha5252** [comment]
Video of what I get on NVDA Win/Chrome

[TinyTake30-07-2019-07-23-21.zip](https://github.com/topcoder-platform/community-app/files/3446718/TinyTake30-07-2019-07-23-21.zip)

**Comment by r0hit-gupta** [comment]
@tosha5252 @lakshmiathreya working for me on Windows/Chrome using NVDA. 

Test URL - https://www.topcoder.com/challenges

**Comment by gets0ul** [comment]
@tosha5252 
Please re-read the issue description carefully.
The expected result is, I quoted:

> The NVDA should read link in sub-menu when user try to mouse hover on them

The requirement is to read the sub-menu item when hovering on it.

Not primary header menu.
What you did in your video is hovering on primary header menu.

Please watch my uploaded video here https://github.com/topcoder-platform/community-app/issues/2836#issuecomment-516340640

It was done on Chrome, Windows 10, NVDA

**Comment by lakshmiathreya** [comment]
My observation during my test is as per your comments @gets0ul @r0hit-gupta. Moving to Pass.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097133<br/><br/>```This is an automated message for crazyk via Topcoder X```
